### ISSUE TITLE
Broken compatibility with Safari on version 0.10.3.0

### ISSUE THREAD

**Issue by joaomlneto** [cs01/gdbgui#164]
##### Describe the issue
Broken compatibility with Safari on version 0.10.3.0 (this issue didn't exist on 0.10.2.0).

When you open gdbgui on Safari 11, the top-most *bar* is missing (the one with the run/step/next/... controls).

##### Steps to replicate

* Upgrade to *latest* version on Debian: `pip install gdbgui --upgrade`
* Run remotely `gdbgui -r ./myApp`
* Open remotely on Safari 11
* Check the top bar (with the step/continue/... controls) is missing

##### Describe your environment
* Operating system and version: Debian Jessie (kernel version 3.16.53 with a minor patch from myself)
* gdbgui version (`gdbgui -v`): `0.10.3.0`
* gdb version (`gdb -v`): `GNU gdb (Debian 7.7.1+dfsg-5) 7.7.1`
* browser: Safari 11.0.2 on macOS 10.13.2
* python packages (`pip freeze`): 
```
Flask==0.12.2
Flask-Compress==1.4.0
Flask-SocketIO==2.9.3
Jinja2==2.10
MarkupSafe==1.0
PAM==0.4.2
Pillow==2.6.1
Pygments==2.2.0
SOAPpy==0.12.22
Werkzeug==0.14.1
argparse==1.2.1
cffi==0.8.6
chardet==2.3.0
click==6.7
colorama==0.3.2
cryptography==0.6.1
defusedxml==0.4.1
docutils==0.12
gdbgui==0.10.3.0
gevent==1.2.2
greenlet==0.4.12
html5lib==0.999
itsdangerous==0.24
lxml==3.4.0
ndg-httpsclient==0.3.2
numpy==1.8.2
pexpect==3.2
ply==3.4
pyOpenSSL==0.14
pyasn1==0.1.7
pycparser==2.10
pycups==1.9.63
pycurl==7.19.5
pygdbmi==0.8.2.0
pygobject==3.14.0
pyinotify==0.9.4
pysmbc==1.0.15.3
python-apt==0.9.3.12
python-debian==0.1.27
python-debianbts==1.11
python-engineio==2.0.2
python-socketio==1.8.4
pyxdg==0.25
reportbug==6.6.3
requests==2.4.3
roman==2.0.0
six==1.11.0
urllib3==1.9.1
wheel==0.24.0
wsgiref==0.1.2
wstools==0.4.3
```

**Comment by joaomlneto** [comment]
No problem! :) I'm actually thankful for `gdbgui`: keep up the good work!

Works perfectly in Chrome!

**Comment by cs01** [comment]
I've never actually tried to open in safari until today, and I was not able to get it to work in the latest version or any version back to 10.0.2.

**Comment by joaomlneto** [comment]
Works OK for me, minus the top bar.

In `0.10.0.2` the bar on the bottom where you type gdb commands would overlap with the output (the last line wouldn't be visible).

Now, it seems to have a huge gap. :)

<img width="1920" alt="screen shot 2018-01-29 at 17 44 49" src="https://user-images.githubusercontent.com/1539767/35525414-4a5b9640-051c-11e8-997e-c220abb0fa16.png">

**Comment by DickvdBrink** [comment]
Same for me on chrome 57 (debian 8) tested with both gdbgui version 0.9.1.0 and also with version 0.10.3

**Comment by cs01** [comment]
@DickvdBrink have you tried w/ a later version of Chrome? It is working for me on Chrome 63.0.3239.132.

**Comment by cs01** [comment]
This has been fixed in v0.11.1.1.
